vm-image-spec.yaml

```diff
@@ -312,6 +312,12 @@ files:
         query: |
           SELECT checkpoints_timed FROM pg_stat_bgwriter;
 
+      - metric_name: logical_snapshot_files
+        type: guage
+        help: 'Number of snapshot files in pg_logical/snapshot'
+        values: [num_files]
```

Comment:
I wonder, is it also possible to add timeline/tenant id as key via `key_labels`? Because they are persisted, so this is more of a timeline feature, not compute/endpoint

We have timeline/tenant set as GUCs (`neon.tenant_id` and `neon.timeline_id`), not sure what's the easiest way to include them into query results

Reply:
Let me see what I can do!